docs/MAKEFILE.md

```diff
@@ -160,6 +160,10 @@ Lists all known make targets.
 -----
 Compiles the `lnrpc` proto files.
 
+`sample-conf-check`
```

Comment:
nit: incomplete commit message.

Reply:
Not really sure what you meant with incomplete. Change the text to 
> Added an entry for sample-conf-check in MAKEFILE.md.